httpd.c

```diff
@@ -128,6 +129,25 @@ char *request_header(const char *name) {
 // get all request headers
 header_t *request_headers(void) { return reqhdr; }
 
+// Handle escape characters (%xx)
+static void uri_unescape(char *s)
+{
+  char c = 0;
```

Comment:
`c` means `count`?

Reply:
c for character (change in chr, which should be clearer).